packages/router/src/directives/router_outlet.ts

```diff
@@ -140,7 +147,18 @@ export class RouterOutlet implements OnDestroy, OnInit {
     // Calling `markForCheck` to make sure we will run the change detection when the
     // `RouterOutlet` is inside a `ChangeDetectionStrategy.OnPush` component.
     this.changeDetector.markForCheck();
-    this.activateEvents.emit(this.activated.instance);
+
+    // If we have an empty outlet component, we want to instead proxy activate/deactivate events
+    // from the inner outlet
+    const c = this.activated.instance;
+    if (c instanceof ɵEmptyOutletComponent) {
+      const sink = new Subscription;
```

Comment:
```suggestion
      const sink = new Subscription();
```

nit: Never invoke a constructor in a `new` statement without using parentheses `()`.
https://google.github.io/styleguide/jsguide.html#disallowed-features-omitting-parents-with-new